.gitignore

```diff
@@ -1,2 +1,158 @@
 
+# Created by https://www.toptal.com/developers/gitignore/api/macos,swift,xcode,carthage,cocoapods,swiftpm
+# Edit at https://www.toptal.com/developers/gitignore?templates=macos,swift,xcode,carthage,cocoapods,swiftpm
+
```

Comment:
this has been forgotten

Reply:
I think there are differences. Also we better not have any other website references if we don't have to